### ISSUE TITLE
Module Real-Dice from TheRipper93 Toggle Button Does Not Function

### ISSUE THREAD

**Issue by Jonoth777** [Dorako/pf2e-dorako-ux#21]
Foundry V11 315
PF2 V5.14.3
Browser: Chrome - 122.0.6261.129 (Official Build) - Latest at time of writing

Modules Enabled:
libwrapper
Real Dice - Manual Rolling
PF2e Dorako UI
PF2e Dorako UX

The Real-Dice module has a button that shows up when "Adjust Chat Controls" is enabled but the button does not function. I've paired down the modules to only yours, real dice, and libwrapper (a dependency for real dice). 

Disabling "Adjust Chat Controls"  resolves the issue.

**Comment by Dorako** [comment]
Yeah, it's the same for Chat Reactions and possibly other modules as well. Dorako UX is sorta removing the original button and adding a new one in a consistent html structure, but unfortunately some of the event handlers don't get copied over.
I've yet to find a proper way to do it, and I'm unlikely to spend a lot of time on it, considering a workaround exists.
If anybody knows the best way to change the html element into something else, but keeping all the listeners, I'd love to know.

**Comment by MrVauxs** [comment]
See if this works now after `v1.3.6`.
